examples/forall-param-reductions.cpp

```diff
@@ -41,118 +41,124 @@ constexpr int HIP_BLOCK_SIZE = 256;
-//
-//
-
-//
-// Define index range for iterating over a elements in all examples
-//
+  //
+  // Note: with this data initialization scheme, the following results will
+  //       be observed for all reduction kernels below:
+  //
+  //  - the sum will be zero
+  //  - the min will be -100
+  //  - the max will be 100
+  //  - the min loc will be N/2
+  //  - the max loc will be N/2 + 1
+  //
+  //
+
+  //
+  // Define index range for iterating over a elements in all examples
+  //
   // _reductions_range_start
   RAJA::TypedRangeSegment<int> arange(0, N);
   // _reductions_range_end
 
-//
-// Define ValLoc Type
-//
+  //
+  // Define ValLoc Type
+  //
 
   using VALLOC_INT = RAJA::expt::ValLoc<int>;
-//----------------------------------------------------------------------------//
+  //----------------------------------------------------------------------------//
 
   std::cout << "\n Running RAJA sequential reductions...\n";
 
   // _reductions_raja_seq_start
-  using EXEC_POL1   = RAJA::seq_exec;
+  using EXEC_POL1 = RAJA::seq_exec;
 
   int seq_sum = 0;
   int seq_min = std::numeric_limits<int>::max();
   int seq_max = std::numeric_limits<int>::min();
   VALLOC_INT seq_minloc(std::numeric_limits<int>::max(), -1);
   VALLOC_INT seq_maxloc(std::numeric_limits<int>::min(), -1);
 
-  RAJA::forall<EXEC_POL1>(host_res, arange,
-    RAJA::expt::Reduce<RAJA::operators::plus>(&seq_sum),
-    RAJA::expt::Reduce<RAJA::operators::minimum>(&seq_min),
-    RAJA::expt::Reduce<RAJA::operators::maximum>(&seq_max),
-    RAJA::expt::Reduce<RAJA::operators::minimum>(&seq_minloc),
-    RAJA::expt::Reduce<RAJA::operators::maximum>(&seq_maxloc),
-    RAJA::expt::KernelName("RAJA Reduce Seq Kernel"),
-    [=](int i, int &_seq_sum, int &_seq_min, int &_seq_max, VALLOC_INT &_seq_minloc, VALLOC_INT &_seq_maxloc) {
-      _seq_sum += a[i];
-
-      _seq_min = RAJA_MIN(a[i], _seq_min);
-      _seq_max = RAJA_MAX(a[i], _seq_max);
-
-      _seq_minloc = RAJA_MIN(VALLOC_INT(a[i], i), _seq_minloc);
-      _seq_maxloc = RAJA_MAX(VALLOC_INT(a[i], i), _seq_maxloc);
-      //_seq_minloc.min(a[i], i);
-      //_seq_maxloc.max(a[i], i);
-      // Note : RAJA::expt::ValLoc<T> objects provide min() and max() methods
-      //        that are equivalent to the assignments with RAJA_MIN and RAJA_MAX
-      //        above.
-    }
-  );
+  RAJA::forall<EXEC_POL1>(
+      host_res,
+      arange,
+      RAJA::expt::Reduce<RAJA::operators::plus>(&seq_sum),
+      RAJA::expt::Reduce<RAJA::operators::minimum>(&seq_min),
+      RAJA::expt::Reduce<RAJA::operators::maximum>(&seq_max),
+      RAJA::expt::Reduce<RAJA::operators::minimum>(&seq_minloc),
+      RAJA::expt::Reduce<RAJA::operators::maximum>(&seq_maxloc),
+      RAJA::expt::KernelName("RAJA Reduce Seq Kernel"),
+      [=](int i,
+          int &_seq_sum,
+          int &_seq_min,
+          int &_seq_max,
+          VALLOC_INT &_seq_minloc,
+          VALLOC_INT &_seq_maxloc) {
+        _seq_sum += a[i];
```

Comment:
This seems odd. Is there a way to make it indent from the beginning of the line containing the opening curly brace?